### PR TITLE
HTML & CSS Webpage 

### PR DESCRIPTION
**Pull request by thecodechaser**
The branch includes the following features for the project:
-Linters config files
-HTML and CSS files
-Updated README files with project details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (dasileker) [inline README.md]: - [ ] You can remove these steps for now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+[Live Demo Link](https://livedemo.com)
+
+Live Demo link will be updated after this project is approved by Code Reviewer Tseam
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
```
Comment (dasileker) [inline README.md:27]: - [ ] Add details on how to download the repo and how you can go to the file and open it.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @thecodechaser,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **To Highlight**
- The linters are passing 👍.
- The commit is good.

## Required Changes ♻️
_Check the comments under the review._

- [ ] The `PR` should contain details on the project, please make it more descriptive.

hhh



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fernandorpm — APPROVED** [review]
# :heavy_check_mark: Pull Request Approved! :heavy_check_mark:
WOW WHAT IS THIS PERFECTION??????? WERE HUMANS SUPPOSED TO BEHOLD SUCH POWER????

Congratulations on your hard work! Please remember to merge the project and don't forget that you're awesome! :heart:



> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!

___
Cheers and Happy Coding! :heart:

> *As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

**Pull request merged**